### PR TITLE
Fix Sonarcloud issues

### PR DESCRIPTION
**Pull request by tvararu**
See commits.

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -2,7 +2,7 @@
 
 # Make sure RUBY_VERSION matches the Ruby version in .ruby-version and Gemfile
 ARG RUBY_VERSION=3.3.0
-FROM ruby:$RUBY_VERSION-slim as base
```
Comment (misaka) [inline Dockerfile:5]: 🤦

File: Dockerfile [hunk 2]
```diff
@@ -18,26 +18,27 @@ RUN gem update --system --no-document && \
 
 
 # Throw-away build stage to reduce size of final image
-FROM base as build
```
Comment (misaka) [inline Dockerfile:21]: 🤦

File: Dockerfile [hunk 3]
```diff
@@ -18,26 +18,27 @@ RUN gem update --system --no-document && \
...
+FROM base AS build
 
 # Install packages needed to build gems and node modules
 RUN apt-get update -qq && \
-    apt-get install --no-install-recommends -y build-essential curl git libpq-dev node-gyp pkg-config python-is-python3
```
Comment (misaka) [inline Dockerfile:25]: This and the quoting below are ++good

File: spec/components/app_card_component_spec.rb [hunk 4]
```diff
@@ -10,7 +10,7 @@
   before { render_inline(component) { body } }
 
   it { should have_css(".nhsuk-card") }
-  it { should have_css("h2.nhsuk-card__heading", text: "A Heading") }
```
Comment (misaka) [inline spec/components/app_card_component_spec.rb:13]: Did Sonarcloud catch this??

**Review by misaka — APPROVED** [review]

File: spec/components/app_card_component_spec.rb (hunk 4 above)
Comment (tvararu) [reply spec/components/app_card_component_spec.rb:13]: Yeah lol "duplicated string"

**Pull request merged**